src/dream.ml

```diff
@@ -80,3 +82,12 @@ let log =
   Dream__middleware.Log.convenience_log
 
 include Dream__middleware.Tag
+
+let ( <.> ) f g = fun x -> f (g x)
```

Comment:
Nit, but this seems unnecessary (and less clear) to define just one function using this operator only twice. Unless you foresee using this a lot in this file in the future, I suggest to just define `now` below explicitly as a function of one argument, with the nested applications written out, and remove this operator.